NXBT-2278 updating ffmpeg

andreinechaev opened the pull request.

nuxeojenkins commented:
[View issue in JIRA](https://jira.nuxeo.com/browse/NCO-502)

jcarsique reviewed (changes requested):
:+1: but edit the commit message accordingly to https://doc.nuxeo.com/corg/git-usage/#main-rules-and-good-practices :

    NXBT-2278: update Ffmpeg to 3.4.2
    
    Add options --enable-libfaac --enable-libxvid --enable-hardcoded-tables libxvidcore-dev

Commit:
NXBT-2278 update FFmpeg to 3.4.2
Add options --enable-libfaac --enable-libxvid --enable-hardcoded-tables libxvidcore-dev
dropped depricated flags and xvidcore added as a dependency
